google/provider/provider_mmv1_resources.go

```diff
@@ -519,8 +519,8 @@ var handwrittenIAMDatasources = map[string]*schema.Resource{
 
 // Resources
 // Generated resources: 596
-// Generated IAM resources: 297
-// Total generated resources: 893
+// Generated IAM resources: 300
+// Total generated resources: 896
```

Comment:
@c2thorn I think the IAM resources are only added to beta. Can you verify if the number change is intended?

Reply:
yes they were set to `min_version: beta`. we may have an error with this logic. I'll take a look.